ledger/error/src/main/scala/com/daml/error/ContextualizedErrorLogger.scala

```diff
@@ -20,6 +19,18 @@ trait ContextualizedErrorLogger {
   def warn(message: String, throwable: Throwable): Unit
   def error(message: String): Unit
   def error(message: String, throwable: Throwable): Unit
+
+  /** Formats the context as a string for logging */
+  protected[error] def formatContextAsString(contextMap: Map[String, String]): String = {
```

Comment:
Moved from ErrorCode

Reply:
I would move it to static object, as this has nothing to do with an instance

Reply:
Ok, done